viz.yaml

```diff
@@ -70,7 +70,7 @@ fetch:
     start.date: "2016-10-5"
     depends: ["sites-data","precip-data"]
     reader: rds
-    refetch: true
+    refetch: FALSE
```

Comment:
If we're going to be rebuilding it for updates, yes.  Although it's unclear if it's actually working.